runtime.md

```diff
@@ -72,16 +75,17 @@ In particular, the state MUST be serialized as JSON.
 
 This operation MUST generate an error if it is not provided a path to the bundle and the container ID to associate with the container.
 If the ID provided is not unique across all containers within the scope of the runtime, or is not valid in any other way, the implementation MUST generate an error and a new container MUST not be created.
-Using the data in [`config.json`](config.md), that is in the root of the bundle's directory, this operation MUST create a new container.
-This includes creating, or entering, the namespaces specified in the [`config.json`](config.md), resource limits, etc and configuring the appropriate capabilities for the container.
-If the `config.json` specifies that a PID namespace is to be created then one MUST be created, but the user-specified code within that namespace MUST NOT be created at this time.
+Using the data in [`config.json`](config.md), this operation MUST create a new container.
+This includes creating, or entering, the namespaces specified in the [`config.json`](config.md), resource limits, etc. and configuring the appropriate capabilities for the container.
+If the `config.json` specifies that a PID namespace is to be created then one MUST be created, but the user-specified code within that namespace MUST NOT be run at this time.
 In some implementations this means that a temporary process is created in the PID namespace but it pauses until the `start` operation is invoked before replacing the process with the user-specified code.
 
 ### Start
 `start <container-id>`
 
 This operation MUST generate an error if it is not provided the container ID.
-This operation MUST create, and start, the user-specified code as specified by the [`process`](config.md#process-configuration) file otherwise an error MUST be generated and no process MUST be started.
+Attempting to start a container that does not exist MUST generate an error.
```

Comment:
If we're listing this for every operation (it seems obvious to me, but I don't mind saying it explicitly), we'll want a similar entry for the “Query State” section.